low-level/files/opt/seagate/sspl/bin/sspl_test

```diff
@@ -1,35 +1,89 @@
-#!/bin/bash -e
+#!/bin/env python3
 
 # Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
-#
-# This program is free software: you can redistribute it and/or modify it under the
-# terms of the GNU Affero General Public License as published by the Free Software
-# Foundation, either version 3 of the License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful, but WITHOUT ANY
-# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
-# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License along
-# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
-# about this software or licensing, please email opensource@seagate.com or
-# cortx-questions@seagate.com.
-
-# Stop the script if any command fails
-set -e -u -o pipefail
-BASE_DIR=$(dirname $0)
-source $BASE_DIR/constants.sh
```

Comment:
constants.sh alternative not needed in python version. as discussed, if needed make use of exising sspl_constants.py

Reply:
yes.. using sspl_constant.py in the python script.